public/content/documentation/native/controls/dropdown.md

```diff
@@ -0,0 +1,337 @@
+## General Notes
+
+How to test a dropdown
+
+## Videos
+
+### iOS Voiceover
+<video controls>
+  <source src="media/video/native/dropdown/dropdown-iOSVoiceOver.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Android Talkback
+<video controls>
+  <source src="media/video/native/dropdown/dropdown-AndroidTalkback.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+## Condensed
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a dropdown
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab, arrow keys or ctl+tab: Focus visibly moves to the Dropdown
+   - Spacebar: Selects and opens the Dropdown on iOS and Android
+   - Enter: Selects and opens the Dropdown on Android
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves to the element, expresses its name, role, value & state (expanded or collapsed)
+   - Doubletap: Selects and opens Dropdown
+
+3. Listen to screenreader output on all devices
+
+   - Name: Purpose is clear and matches any visible label
+   - Role: Identifies itself as a button in iOS and "double tap to activate" in Android
+   - Group: Visible label is grouped or associated with the Dropdown in a single swipe
+   - State: Expresses its state (disabled/dimmed). Expanded/collapsed states are announced on the elements that close or open the dropdown
+
+4. Test device settings
+
+   - Text resize: Text can resize up to 200% without losing information
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/dropdown](https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/dropdown)
+
+## Gherkin
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a dropdown
+
+GIVEN THAT I am on a screen with a dropdown
+
+1. Scenario: Test keyboard actions
+
+   - WHEN I press the "TAB", "ARROW KEYS", or "CTRL+TAB" keys 
```

Comment:
The spacing on the first scenario of gherkin has different spacing than the rest of the gherkin. It might be resolved by removing the spaces between "THEN" and "WHEN" statements?